README.md

```diff
@@ -13,30 +13,34 @@ See the following tables for all of the current FreeRTOS CMSIS packs available.
-| Backoff Algorithm            | https://github.com/FreeRTOS/backoffAlgorithm/                      |[4.0.0(1.0.0)](https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/AWS.backoffAlgorithm.4.0.0.pack) (latest)<br>[1.0.0-Beta(1.0.0)](https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/AWS.backoffAlgorithm.1.0.0-Beta.pack)|
+| FreeRTOS+TCP                | https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/                     |[5.0.0(3.1.0)](https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/AWS.FreeRTOS-Plus-TCP.5.0.0.pack) (latest)<br> [4.0.1(2.3.2-LTS-Patch-2)](https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/AWS.FreeRTOS-Plus-TCP.4.0.1.pack) <br>[4.0.0(2.3.2-LTS-Patch-2)](https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/AWS.FreeRTOS-Plus-TCP.4.0.0.pack)<br>[2.3.2-Beta(2.3.2)](https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/AWS.FreeRTOS-Plus-TCP.2.3.2-Beta.pack)
+| Backoff Algorithm           | https://github.com/FreeRTOS/backoffAlgorithm/                      |[4.1.0(1.3.0)](https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/AWS.backoffAlgorithm.4.1.0.pack) (latest)<br> [4.0.0(1.0.0)](https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/AWS.backoffAlgorithm.4.0.0.pack) <br>[1.0.0-Beta(1.0.0)](https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/AWS.backoffAlgorithm.1.0.0-Beta.pack)|
 
 ### FreeRTOS LTS
 | Library Name                | Repository URL                                                     | Available Packs<br>\<Pack Version\>(\<Library Version\>)|
 |-----------------------------|--------------------------------------------------------------------|---------------------------------------------------------|
-| FreeRTOS Long Term Support  | https://github.com/FreeRTOS/FreeRTOS-LTS/                          |[2021.0.1(202012.04-LTS)](https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/AWS.FreeRTOS_LTS.2021.0.1.pack) (latest)<br>[2021.0.0(202012.04-LTS)](https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/AWS.FreeRTOS_LTS.2021.0.0.pack)<br>[2020.12.1-Beta(202012.01-LTS)](https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/AWS.FreeRTOS_LTS.2020.12.1-Beta.pack)|
+| FreeRTOS Long Term Support  | https://github.com/FreeRTOS/FreeRTOS-LTS/                          |[2022.0.0(202210.01-LTS)](https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/AWS.FreeRTOS_LTS.2022.0.0.pack) (latest)<br> [2021.0.1(202012.04-LTS)](https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/AWS.FreeRTOS_LTS.2021.0.1.pack) <br>[2021.0.0(202012.04-LTS)](https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/AWS.FreeRTOS_LTS.2021.0.0.pack)<br>[2020.12.1-Beta(202012.01-LTS)](https://freertos-cmsis-packs.s3.us-west-2.amazonaws.com/AWS.FreeRTOS_LTS.2020.12.1-Beta.pack)|
 
```

Comment:
Verified updated links have correct versions and can download from all of them.

Reply:
Thanks!!